core/foundation/src/TClassEdit.cxx

```diff
@@ -1871,8 +1871,10 @@ class NameCleanerForIO {
       if (!mother) return false;
       bool isSTLContOrArray = true;
       while (nullptr != mother){
-         auto stlType = TClassEdit::IsSTLCont(mother->fName+"<>");
-         isSTLContOrArray &= ROOT::kNotSTL != stlType || TClassEdit::IsStdArray(mother->fName+"<");
+         auto tmpStr = mother->fName+"<>";
+         auto stlType = TClassEdit::IsSTLCont(std::string_view(tmpStr.data(), tmpStr.size()));
```

Comment:
I still don't understand why we need to provide explicit casts to `string_view` for `IsSTLCont`, `IsStdArray` etc. Isn't passing `tmpStr` as argument enough?